### DISCUSSION TITLE
Automatic feed refresh and email notifications

### DISCUSSION THREAD

**Discussion by xNiika** [FreshRSS/FreshRSS#6670] [Q&A]
Is it possible to refresh the feed automatically every x minutes or hours and get email notifications about new articles?

**Comment by Alkarex** [comment]
Hello,
At the moment, there is no such notification mechanism built in. However, you can [produce an RSS feed with any search of interest](https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries), and point to that RSS feed from a third party alerting tool, for instance https://ifttt.com/connect/feed/if_notifications

See also https://github.com/FreshRSS/FreshRSS/discussions/6480
